### PR TITLE
More intuitive UI/UX

### LINKED ISSUE #407: Layer split / variable comparison mode is confusing
The current implementation of the split layer feature confuses some users, because the 2nd variable of two variables to be compared is displayed on the left side. This decision has been made to continue supporting the interactive color legend located on the right side for the currently selected variable. 

Since it is now possible to display interactive color legends on both sides (#401), we have at least two options to  avoid or mitigate the confusion:

1. place the first variable on the 1st side, and the 2nd variable on the right side.
2. avoid the term _second_ and the `(#2)` label suffix in the UI and use the term _pinned_ or _fixed_ instead. The pinned variable stays left, the currently selected variable stays right.

To me, option 2 seems most intuitive.

### PR DESCRIPTION
**Pull request by forman**
Avoiding confusion regarding variable comparison. 

Improved the user experience in several ways:
- No longer using the term _second variable_ and the `(#2)` suffix in variable titles. Instead, it is now the _pinned variable_. A pin icon is now used to mark a pinned variable.
- Moved map-related actions directly into the map, namely
  - show/hide map layer panel (moved from main toolbar)
  - enable/disable map split mode (moved from layer control panel)
  - enable/disable new map info box  (moved from layer control panel)

Closes #407

**EDIT**

- Adjusted map tooltip style, added new tooltips.
- Improved the visibility of the layer split bar over bright map backgrounds.
- 2nd color legend is no longer shown if split mode is off

### CODE REVIEW

File: CHANGES.md
Comment (clarasb) [inline CHANGES.md:1]: The new button for the direct generation of statistics (`Add Statistics`) should also be included here.

File: src/components/MapControlActions.tsx
Comment (clarasb) [inline src/components/MapControlActions.tsx:1]: Added Tooltip titles for the 2nd and 3rd button would be great.

Comment (AliceBalfanz) [reply src/components/MapControlActions.tsx:1]: I second this

**Comment by AliceBalfanz** [comment]
nothing happens when I select 'enable/disable new map info box (moved from layer control panel)', where should I find the change when I toggle that button?

File: CHANGES.md
Comment (forman) [reply CHANGES.md:1]: Done.

**Comment by forman** [comment]
> nothing happens when I select 'enable/disable new map info box (moved from layer control panel)', where should I find the change when I toggle that button?

Move mouse over map!

**Comment by forman** [comment]
> * if comparison mode is switched off again after it was once active, then parts of the left ColorBar remains in the top left corner, but only for variable names with overlength 

Done.

>related to [Incorrect scaling of ColorBar for variables with long titles #410](https://github.com/xcube-dev/xcube-viewer/issues/410)

Done: #414

**Comment by AliceBalfanz** [comment]
> > nothing happens when I select 'enable/disable new map info box (moved from layer control panel)', where should I find the change when I toggle that button?
> 
> Move mouse over map!

Ah, ok!

**Review by AliceBalfanz — APPROVED** [review]
I like the new changes a lot! But tool tips as Clara mentioned  for the all of the icons that were moved on the map would be great :)

**Pull request merged**